examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -61,8 +61,8 @@
 # Define the dimension reduction to be used.
 # Here we use a classical univariate feature selection based on F-test,
 # namely Anova. We set the number of features to be selected to 500
-from sklearn.feature_selection import SelectKBest, f_classif
-feature_selection = SelectKBest(f_classif, k=500)
+from sklearn.feature_selection import SelectPercentile, f_classif
+feature_selection = SelectPercentile(f_classif, k=500)
```

Comment:
That seems strange to me: 500 is not a valid percentile: percentiles and between 0 and 100. 500 voxels is something like 5% of the brain, so I would have expected 5 here. In addition, the valid argument of SelectPercentile is "percentile" and not "k". Are you sure that this example actually runs OK?